README.md

```diff
@@ -0,0 +1,16 @@
+# Endless OS Desktop Extension
+
+Provides the signature Endless OS desktop experience atop GNOME Shell 41 or 42.
+
+## Differences from upstream GNOME Shell
+
+- App grid on the desktop
+- Applications button added to top bar to quickly show the app grid
+- Auto-hiding dash outside the Activities Overview
+- Tweaked workspace styling
+
+## Install
+
+Compatible with GNOME 41 and 42.
+
+This extension is distributed via [extensions.gnome.org](https://extensions.gnome.org/extension/5419/endless-desktop/); head there or use the excellent [Extension Manager](https://flathub.org/apps/details/com.mattjakeman.ExtensionManager) to install it.
```

Comment:
This is certainly an improvement over no README. However the version on ego is likely to drift out of date – there is no automation to push new versions of the extension there. Might be worth a note about how to install from source?